src/components/ErrorBoundary/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import style from "./styles.module.scss";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const ErrorBoundary = (props: any) => {
```

Comment:
Are you sure this Error boundary is implemented in correct way ? I was able to break your app and didn't see default page. https://reactjs.org/docs/error-boundaries.html